### ISSUE TITLE
Uncanny behavior on Brave: middleclick opens link and closes a random tab

### ISSUE THREAD

**Issue by matto-in-cor** [rxhanson/Middle-Community#59]
Middleclick used to work fine, but in Brave it has a strange behavior. When I clic on a link to open it on a new tab, Brave closes another tab. It seems, but I'm not sure, that has something to do with tab groups. [Brave Version 1.47.186 Chromium: 109.0.5414.119 (Official Build) (arm64)](https://brave.com/latest/) MacOS Ventura 13.0.1 (22A400)

**Comment by rxhanson** [comment]
Thanks for reporting. Can you send a screenshot of your Middle preferences or let me know what you have configured to act as a middle click? There are slight differences with how the different gestures work to register middle clicks.

Unfortunately, it's likely that this would have to be fixed Brave/Chromium side, as all Middle is doing is registering middle clicks.

**Comment by matto-in-cor** [comment]
![image](https://user-images.githubusercontent.com/124456358/216781272-df34d938-8183-41e5-9ba0-201aafa69853.png)
![image](https://user-images.githubusercontent.com/124456358/216781276-3e7c6094-03ab-4ba9-92f7-80fb38c2bb68.png)
I also tried disabling tab management extensions but nothing changed. 

https://user-images.githubusercontent.com/124456358/216781938-c67b447d-8a87-4ac3-8931-57873a1eb351.mp4

**Comment by rxhanson** [comment]
Thanks for the additional info. I tested this out on my own and was unable to reproduce it. 

Just to isolate the issue to Brave, do things work fine in other browsers? 

I think the proper way to go about it is to report it to Brave. Since I cannot reproduce it, this is best reported by you. I don't mind adding details to the issue once it is created if necessary.

[Submitting a bug to the Brave team](https://support.brave.com/hc/en-us/articles/360018228951-How-do-I-submit-a-bug-report-#:~:text=The%20best%20way%20to%20do,Brave%20Team%20staff%20to%20see.&text=It's%20built%20to%20address%20many%20common%20questions%20and%20known%20issues.)

**Comment by matto-in-cor** [comment]
Try with the window in fullscreen mode, I noticed that in the normal window mode it works fine.

**Comment by rxhanson** [comment]
Hmm, I still can't reproduce this in fullscreen either. Does it happen with Middle configured for three finger click as well? Any chance you could try with a mouse with a physical middle button, without Middle running at all?

**Comment by matto-in-cor** [comment]
It makes no difference using the three finger click configuration, but if the fn+left click options is enabled and i fn+left click a link it works correctly!

**Comment by rxhanson** [comment]
Do you have any other middle clicking applications running, like BetterTouchTool?

**Comment by matto-in-cor** [comment]
No, I had installed that (perhaps years ago). Previously I installed the open source version I fount on GitHub but at a certain point it stopped working so I switched to this paid version.

... Ok, I checked in the activity manager and I found that there was an active as a process called "middleclick". I guess this was the problem, good work! I'm going to uninstall that, I bet this will solve the issue. Many thanks!
